python/dgl/graphbolt/internal/utils.py

```diff
@@ -120,3 +121,25 @@ def get_attributes(_obj) -> list:
         and not callable(getattr(_obj, attribute))
     ]
     return attributes
+
+
+def read_edges(dataset_dir, edge_fmt, edge_path):
```

Comment:
add dedicated testcase for unsupported format and capture exception with matched log.